src/jtag3.c

```diff
@@ -1649,6 +1649,15 @@ static int jtag3_open_updi(PROGRAMMER * pgm, char * port)
 {
   avrdude_message(MSG_NOTICE2, "%s: jtag3_open_updi()\n", progname);
 
+  LNODEID ln;
```

Comment:
Added this section here to output the hvupdi_support list when using _-vv_. Should it be put somewhere else?

Reply:
I think it's fine!